tests/dialects/test_llvm.py

```diff
@@ -68,16 +68,16 @@ def test_llvm_getelementptr_op_invalid_construction():
 
     # check that passing an opaque pointer to GEP without a pointee type fails
     with pytest.raises(ValueError):
-        llvm.GEPOp.get(opaque_ptr, llvm.LLVMPointerType.typed(builtin.i32),
-                       [1])
+        llvm.GEPOp.get(opaque_ptr, [1],
```

Comment:
I would pre-assign [1] to some variable "const_indices"? and pass the var so as to "feel clearer"

Reply:
I changed it to a keyword argument, that should improve readability